docs/docs/howto/installation/run-source.rst

```diff
@@ -7,25 +7,33 @@ Klaw is a Java project. Follow the below steps to run this project from the sour
 
    .. tip:: You can also clone repository to use the "bleeding edge" version if you wish.
 
-2. Configure Cluster Api access
+2. Experimental UI Configuration
+   A redesigned user interface for Browse topics, providing a new look and feel for managing Kafka topics. It consists of changes to the layout, styling,
+   and overall appearance of the interface, as well as improvements to make browsing and working topics easier and more efficient. This new look and feel
```

Comment:
...not to sound like  a broken record here, but we _could_ do:

"...as well as improvements to make browsing and working topics easier, more efficient and accessible."

Because we did do work there, so, why not tell the 🌐 ?

Reply:
Updated. Pls take another look.

Reply:
Love it! :))))))